linux/keyman-config/com.keyman.gschema.xml

```diff
@@ -1,8 +1,12 @@
 <schemalist>
   <schema id="com.keyman.options" path="/desktop/ibus/keyman/options/">
-
     <child name="keyboard-options" schema="com.keyman.options.child"/>
 
+    <key name="error-reporting" type="b">
+        <default>true</default>
+        <summary>Automatic error reporting</summary>
+        <description>true to enable automatic reporting of errors to Sentry</description>
```

Comment:
Will these be localizable?

Reply:
Currently not. [It seems to be possible to add localization for schemas](https://docs.gtk.org/gio/class.Settings.html), but I can't find details how to do that. And I think the important part is that it's localized in the UI - if a user wants to deal directly with the settings I assume he has some knowledge of English.

Reply:
Righto -- I missed that the UI is localizable already and these were internal strings. All good :)